### PR TITLE
SeriesPanel: add publisher and publication year.

### PR DESCRIPTION
**Pull request by Pineirin**
Set the text for no access online's urls.
Set the title for the volumes in the series.
Add a section for the physical volumes.
Fix the display of authors.
closes https://github.com/CERNDocumentServer/cds-ils/issues/538

Screenshot 2022-01-18 at 09 28 25
Screenshot 2022-01-18 at 09 29 51
Screenshot 2022-01-18 at 09 30 11

### CODE REVIEW

**Comment by kprzerwa** [comment]
commit message is copied 3 times, could you fix it ?

File: src/lib/pages/frontsite/Series/SeriesDetails/SeriesPanel/SeriesPanelMobile.js [hunk 1]
```diff
@@ -33,18 +38,35 @@ class SeriesPanelMobile extends Component {
...
               </ILSHeaderPlaceholder>
-              <ILSParagraphPlaceholder linesNumber={1} isLoading={isLoading}>
+              <ILSParagraphPlaceholder linesNumber={2} isLoading={isLoading}>
                 <SeriesAuthors
                   prefix="by "
-                  itemProps={{ as: 'h4' }}
+                  listProps={{ as: 'h4' }}
                   authors={series.metadata.authors}
                 />
+                {(isMultipartMonograph(modeOfIssuance, seriesType) ||
```
Comment (kprzerwa) [inline src/lib/pages/frontsite/Series/SeriesDetails/SeriesPanel/SeriesPanelMobile.js:54]: here as well, I am not sure if I understand this conditional statement

File: src/lib/modules/Series/SeriesAccess.js [hunk 2]
```diff
@@ -2,30 +2,57 @@ import { ILSImagePlaceholder } from '@components/ILSPlaceholder';
...
+          }
+        />
         <br />
         It's not possible to loan an entire series, but you can loan individual
         volumes.
         <br />
         Please see the list of available volumes and issues below.
       </ILSImagePlaceholder>
+      <ILSImagePlaceholder style={{ height: 200 }} isLoading={isLoading}>
+        {isSerialPeriodical(modeOfIssuance, seriesType) &&
```
Comment (kprzerwa) [inline src/lib/modules/Series/SeriesAccess.js:32]: why do you check if it is a periodical ?

File: src/lib/pages/frontsite/Series/SeriesDetails/SeriesPanel/SeriesPanel.js [hunk 3]
```diff
@@ -49,22 +54,40 @@ class SeriesPanel extends Component {
                             : null
                         }
                         TypeOrModeOfIssuance={
-                          series.metadata.series_type ||
-                          series.metadata.mode_of_issuance
+                          isSerialSerial(modeOfIssuance, seriesType)
+                            ? 'BOOK SERIES'
+                            : seriesType || modeOfIssuance
```
Comment (kprzerwa) [inline src/lib/pages/frontsite/Series/SeriesDetails/SeriesPanel/SeriesPanel.js:59]: what will be displayed for multipart monographs ?

**Comment by kprzerwa** [comment]
For the physical volumes section I think we need something more like 
"Where to find" section which we have for the books/articles etc. It does not have to be as complex, but I believe it should be at least placed in the same location in the UI.

**Comment by kprzerwa** [comment]
commit message has still duplicated entries, please fix

**Review by ntarocco — APPROVED** [review]

**Pull request merged**